### DISCUSSION TITLE
Need clarification on SC 1.4.4 Resize Text

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#2865] [Q&A] [answered]
Hello everyone,

I have a scenario where the web page has few features(e.g. Chat, Pools, Notes) in a different area of the web page.
When I increase the zoom level to 200%, a few features are not visible properly. But I can drag and drop the features with the help of a mouse and by changing the position of the features, I can access the features.

Can we consider it as a pass for SC's 1.4.4 Resize Text?

But the point here is that drag-and-drop functionality is not keyboard-operable. Will be it a failure here for 1.4.4 SC?

**Comment by patrickhlauke** [comment, marked as answer]
Sounds like pass for 1.4.4 Resize Text, but failure of 2.1.1 Keyboard, to me
